crates/viewer/re_data_ui/src/item_ui.rs

```diff
@@ -185,8 +185,7 @@ pub fn instance_path_icon(
         if db
             .storage_engine()
             .store()
-            .all_components_on_timeline(timeline, &instance_path.entity_path)
-            .is_some()
+            .entity_has_data_on_timeline(timeline, &instance_path.entity_path)
```

Comment:
This behaves slightly differently for reasons I don't really comprehend, at least in some tests.

Reply:
According to Clement, one is affected by GC but not the other, which doesn't explain why it break a test where no GC is supposed to take place.